src/components/FormErrors/__test__/FormErrors.test.tsx

```diff
@@ -15,4 +16,18 @@ describe('Form Errors', () => {
     const { queryByText } = render(<FormErrors formErrors={errors} />);
     expect(queryByText('There is a problem')).toBeInTheDocument();
   });
+
+  it('should focus text field when error is clicked', () => {
+    const { getByTestId } = render(<FormErrors formErrors={errors} />);
```

Comment:
Instead of doing getByTestId, could you not do: `getByRole('link', { name: '<error message>' })` ?
Minor, but just better practice to use the actual accessible properties

Reply:
Changed to use getByRole